lib/network/modules/InteractionHandler.js

```diff
@@ -165,8 +165,11 @@ class InteractionHandler {
 
   onContext(event) {
```

Comment:
Yes, it's what I need, i'll explain, when user clicks right-click, he does not have access to `hoverNode`'s params, so on right click I would like to access the node/edge that i'm hovering.
:)